### PR TITLE
Remove all trailing spaces from code 

### PR DESCRIPTION
**Pull request by fizray**
Im using codefactor to remove the trailing spaces from all codes maybe some code need some fixes

### CODE REVIEW

**Comment by fizray** [comment]
i hope it doesnt break the code

**Comment by fizray** [comment]
1 Month Aniversary for this pull req

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # cobalt-cluster: A Complete Rewrite of Cobalt Butterfly
+[![CodeFactor](https://www.codefactor.io/repository/github/hecksalmonids/cobalt-cluster/badge)](https://www.codefactor.io/repository/github/hecksalmonids/cobalt-cluster)
```
Comment (CorruptedPhazite) [inline README.md]: Please remove CodeFactor readme badge

**Review by CorruptedPhazite — CHANGES_REQUESTED** [review]
Please remove CodeFactor readme badge

**Comment by fizray** [comment]
wait how to resolve github
Do i need to close this pull and re pull again?

**Review by fizray — COMMENTED** [review]
Re reviewed and changed

File: lib/utils.rb [hunk 2]
```diff
@@ -1,597 +1,597 @@
...
+  alias_method(:pl, :plural)
+
+  # Rudimentary pluralize; returns pluralized str with added 's' only if the given int is not 1
+  # strips out number if 1
+  # @param  [Integer] int the integer to test
+  # @param  [String]  str the string to pluralize
+  # @return [String]  singular form (i.e. squid) if int is 1, plural form (8 squids) otherwise
+  def plural_every(int, str)
+    return "#{int} #{str}s" unless int == 1
+    str.to_s
```
Comment (CorruptedPhazite) [inline lib/utils.rb]: please revert to `"#{str}"`

Comment (fizray) [reply lib/utils.rb]: where should i revert

Comment (CorruptedPhazite) [reply lib/utils.rb]: line 152

File: src/dev/economy_passive.rb [hunk 3]
```diff
@@ -253,6 +253,6 @@ def image?(message)
   # @return [bool] Has link?
   def link?(message)
     return false if message.nil? or message.content.nil?
-    return message.content =~ /#{URI::regexp(['http', 'https'])}/
+    return message.content =~ /#{URI::DEFAULT_PARSER.make_regexp(['http', 'https'])}/
```
Comment (CorruptedPhazite) [inline src/dev/economy_passive.rb]: please revert 256 to `return message.content =~ /#{URI::regexp(['http', 'https'])}/`

Comment (fizray) [reply src/dev/economy_passive.rb]: ok

File: src/dev/economy.rb [hunk 4]
```diff
@@ -63,11 +63,11 @@ def self.get_user_checkin_value(user_id)
     when VERIFIED_ROLE_ID
       role_yaml_id = "checkin_verified"
     when INVALID_ROLE_ID
-      role_yaml_id = "checkin_new"    
+      role_yaml_id = "checkin_new"
     end
 
     if role_yaml_id == nil
-      raise RuntimeError, "Unexpected role ID received, there may be a new role that needs to be accounted for by checkin!"
```
Comment (CorruptedPhazite) [inline src/dev/economy.rb]: please revert line 70 to `raise RuntimeError, "Unexpected role ID received, there may be a new role that needs to be accounted for by checkin!"`

Comment (fizray) [reply src/dev/economy.rb]: ok

File: src/helper/bank.rb [hunk 5]
```diff
@@ -140,7 +140,7 @@ def withdraw(user_id, amount)
 
     # iterate through balances and remove until amount is withdrawn
     user_transactions = USER_BALANCES.where{Sequel.&({user_id: user_id}, (amount > 0))}.order(Sequel.asc(:timestamp))
-    while amount > 0 and user_transactions.count > 0 do
```
Comment (CorruptedPhazite) [inline src/helper/bank.rb]: please revert line 143 to `while amount > 0 and user_transactions.count > 0 do`

File: src/main/basic_commands.rb [hunk 6]
```diff
@@ -65,27 +65,27 @@ module Bot::BasicCommands
 
     # Checks to see if Update_Check_Frequency.txt exists in /scr/ as well as reading the contents of the file. This file is generated via the auto-updater script at startup and is only deleted if +exit is used
 
-    if File.exists? "Update_Check_Frequency.txt"
```
Comment (CorruptedPhazite) [inline src/main/basic_commands.rb]: please revert line 68 to `if File.exists? "Update_Check_Frequency.txt"`

**Review by CorruptedPhazite — CHANGES_REQUESTED** [review]

File: src/helper/bank.rb (hunk 5 above)
Comment (fizray) [reply src/helper/bank.rb]: ok

File: src/main/basic_commands.rb (hunk 6 above)
Comment (fizray) [reply src/main/basic_commands.rb]: ok

**Review by CorruptedPhazite — APPROVED** [review]

**Comment by CorruptedPhazite** [comment]
@Paprika4785  Only code changes removing trailing spaces were approved. Rejected commits (i.e. the CodeFactor badge) may be separately submitted as a pull request for consideration

**Comment by fizray** [comment]
okay

**Pull request merged**